ArjunaJTS/jts/classes/com/arjuna/ats/jts/common/JTSEnvironmentBean.java

```diff
@@ -55,6 +55,15 @@
     @FullPropertyName(name = "com.arjuna.ats.jts.recovery.commitTransactionRetryLimit")
     private volatile int commitedTransactionRetryLimit = 3;
 
+    private volatile int transactionServiceId = 0xDEADBEEF;
```

Comment:
Good point - can you raise a JBTM (once this merges) to change the slotId in 6.later